src/odemis/driver/test/piezomotor_test.py

```diff
@@ -34,6 +34,7 @@
 # Export TEST_NOHW=1 to force using only the simulator and skipping test cases
 # needing real hardware
 TEST_NOHW = (os.environ.get("TEST_NOHW", "0") != "0")  # Default to Hw testing
+TEST_NOHW = 1
```

Comment:
Drop before merge.

Reply:
yes, I converted the whole PR to a draft, since I did not manage to fix it yet. I think I will just close it, since it will be a while before I can work on this again.